components/tabs/tabs.tsx

```diff
@@ -87,7 +87,6 @@ const Tabs: React.FC<React.PropsWithChildren<TabsProps>> = ({
         <style jsx>{`
           .tabs {
             width: initial;
-            overflow: auto;
```

Comment:
The scrolling of the `Tabs.Header` does not require the entire container to be added to `overflow`.